README.md

```diff
@@ -112,11 +112,13 @@ It's simplest to create your configuration file as a "raw" resource file in your
             <data
                 android:scheme="msauth"
                 android:host="<YOUR_PACKAGE_NAME>"
-                android:path="/<YOUR_BASE64_URL_ENCODED_PACKAGE_SIGNATURE>" />
+                android:path="/<YOUR_BASE64_ENCODED_PACKAGE_SIGNATURE>" />
         </intent-filter>
     </activity>
 ```
 
+>NOTE: The package signature required here needs to be base 64 encoded only as opposed to being base 64 url encoded. Please refer to [this FAQ](https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki/MSAL-FAQ#redirect-uri-issues) for more information on common redirect uri issues.
```

Comment:
nit: I think adding "The package signature required here needs to be base 64 encoded only as opposed to being base 64 url encoded" could confuse new developers (without context of #1019). Just leaving the FAQ link here should be good enough IMO.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1022/commits/5e1a0fe5b967c51c44fb9124a1ca40a11e8cbc5f